### PR TITLE
Updates snarkOS to `snarkVM v0.7.5`, removes `snarkos-toolkit`

### PR DESCRIPTION
**Pull request by howardwu**
## Motivation

- Updates snarkOS to `snarkVM v0.7.3`
- Removes `snarkos-toolkit`
- Update CI to use Rust `v1.53.0`

### CODE REVIEW

**Comment by howardwu** [comment]
@niklaslong @ljedrz could you take a look at this PR and update the hardcoded tests with new values?

File: .github/dependabot.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ updates:
   schedule:
     interval: daily
     time: "10:00"
+  target_branch: "staging"
```
Comment (ljedrz) [inline .github/dependabot.yml:8]:
```suggestion
  target-branch: "staging"
```
(as commented in https://github.com/AleoHQ/snarkOS/pull/904)

**Comment by ljedrz** [comment]
@howardwu if I'm not mistaken, the test errors should go away once the test data gets re-generated with the new parameters; I'll take a stab at it :+1:.

**Review by niklaslong — COMMENTED** [review]
Looks good to me though I may have missed something since the PR is quite large and the changes are widespread.

**Review by ljedrz — APPROVED** [review]
LGTM, there's just one small conflict left

**Pull request merged**